Contact/Contact Class and Smaller Related Data Classes/Contact.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Nate.ContactApp
+{
+    public class Contact
+    {
+        // Ideally need a 2-wide list of attributes with only a couple default attributes (ie first/last name)
+        public int ContactID;
+        public bool IsActiveRecord;
+        public string FirstName;
+        public string LastName;
+        public PhoneNumber Home;
```

Comment:
Typically I would just define these as public PhoneNumber Home { get; set; } = new PhoneNumber();  instead of using a constructor.  That being said, defaulting them to null might make more sense as that is more of what I would expect if there isn't this type of phone number.

Reply:
I will make sure they default to null because that behaviour does make sense. I went this route with smaller classes for phone numbers and some other contact attributes in order to keep any logic related to managing that associated with the actual attribute in hopes of making things more readable. Otherwise I think I could make them into structs and accomplish similar goals.

Reply:
Unless you really think should be otherwise I'll leave the Phone Number class as is.